### DISCUSSION TITLE
Image not showing

### DISCUSSION THREAD

**Discussion by binarybehemoth** [vercel/next.js#74327] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Next.js 15.1.2 on StackBlitz
```

### Which example does this report relate to?

https://stackblitz.com/edit/webcodingcenter-nextjs-script-i6qp2vrx?file=app%2Fpage.tsx

### What browser are you using? (if relevant)

Chrome

### How are you deploying your application? (if relevant)

StackBlitz

### Describe the Bug

The following code snippet is not working. The image is stored as '/public/love.png'.

When I switched Next.js to version 14.1.2, it worked.
```
import Image from 'next/image';

export default function Home() {
  return <Image src={'/love.png'} width="300" height="300" alt="love" />;
}
```

### Expected Behavior

An image should be displayed.

### To Reproduce

Click the StackBlitz link.

**Comment by Min-gzm** [comment]
This may be a StackBlitz‘s issue, because I have downloaded your example project on Stackblitz and running it locally - the images display correctly there. Additionally, even basic <img> tags fail to display images on StackBlitz.

**Comment by 2395000206** [comment]
![企业微信截图_17352887261003](https://github.com/user-attachments/assets/018c7982-b77b-4210-b950-be05032d8121)
It can be displayed but an error is reported
